test_sets/Defects/Defects_release_8_1.txt

```diff
@@ -1 +1,3 @@
+./test_scripts/Defects/8_1/1883_1_ChoiceSet_being_processed_in_DeleteInteractionChoceSet.lua
+./test_scripts/Defects/8_1/1883_2_ChoiceSet_being_processed_in_DeleteInteractionChoceSet_after_resumption.lua
```

Comment:
I would rename scripts in way of requirement. E.g.:
> ..._PerformInteraction_REJECTED_for_being_deleted_InteractionChoceSet

Reply:
Done